### ISSUE TITLE
Parameters for finetuning on UCF101 and JHMDB

### ISSUE THREAD

**Issue by jiaozizhao** [deepmind/kinetics-i3d#33]
Hi, May I know what parameters are you used for finetuning on UCF101 and JHMDB? Like learning rate, momentum, weight_decay, and steps. Thanks

**Comment by Sushant-aggarwal** [comment]
have you found any updates on your question??
